Viewing the Data
One of the most used method for getting a quick overview of the DataFrame, is the head() method
The head() method returns the headers and a specified number of rows, starting from the top
Xem dữ liệu

Một trong những phương pháp được sử dụng nhiều nhất để có cái nhìn tổng quan nhanh về DataFrame là phươgn thứg head)
Phương thức head()ức trả về các tiêu đề và số hàng được chỉ định, bắt đầu từ trên cùg.

In [1]:
# Get a quick overview by printing the first 10 rows of the DataFrame:
# Nhận tổng quan nhanh bằng cách in 10 hàng đầu tiên của DataFrame:
import pandas as pd

df = pd.read_csv('C:/Users/thanh/ttc-python/3-Modules/3.2.Pandas/3.2a.P-Toturial/data.csv')

print(df.head(10))

   Duration  Pulse  Maxpulse  Calories
0        60    110       130     409.1
1        60    117       145     479.0
2        60    103       135     340.0
3        45    109       175     282.4
4        45    117       148     406.0
5        60    102       127     300.0
6        60    110       136     374.0
7        45    104       134     253.3
8        30    109       133     195.1
9        60     98       124     269.0


In our examples we will be using a CSV file called 'data.csv'
Download data.csv, or open data.csv in your browse
Trong ví dụ của chúng tôi, chúng tôi sẽ sử dụng tệp CSV có tên 'data.csv

Tải xuống data.csv hoặc mở data.csv trong trình duyệt của br.

Note: if the number of rows is not specified, the head() method will return the top 5 rows
Lưu ý: nếu số lượng hàng không được chỉ định, phương thức head() sẽ trả về 5 hàng trên cùng

In [3]:
# Print the first 5 rows of the DataFrame:
# In 5 hàng đầu tiên của DataFrame:
import pandas as pd

df = pd.read_csv('C:/Users/thanh/ttc-python/3-Modules/3.2.Pandas/3.2a.P-Toturial/data.csv')

print(df.head())

   Duration  Pulse  Maxpulse  Calories
0        60    110       130     409.1
1        60    117       145     479.0
2        60    103       135     340.0
3        45    109       175     282.4
4        45    117       148     406.0


There is also a tail() method for viewing the last rows of the DataFrame
The tail() method returns the headers and a specified number of rows, starting from the bottmo
Ngoài ra còn có mộ phương thứct tail(p để xem các hàng cuối cùng của DataFram

Phươ thứcng tailức trả về các tiêu đề và số hàng được chỉ định, bắt đầu từ dưới cùm.

In [4]:
# Print the last 5 rows of the DataFrame:
# In 5 hàng cuối cùng của DataFrame:
print(df.tail()) 

     Duration  Pulse  Maxpulse  Calories
164        60    105       140     290.8
165        60    110       145     300.0
166        60    115       145     310.2
167        75    120       150     320.4
168        75    125       150     330.4


Info About the Data
The DataFrames object has a method called info(), that gives you more information about the data set
Thông tin về dữ liệu
Đối tượng DataFrames có một phương thức được gọi là info(), cung cấp cho bạn thêm thông tin về tập dữ li..

In [11]:
# Print information about the data:
# In thông tin về dữ liệu:
print(df.info()) 

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 169 entries, 0 to 168
Data columns (total 4 columns):
 #   Column    Non-Null Count  Dtype  
---  ------    --------------  -----  
 0   Duration  169 non-null    int64  
 1   Pulse     169 non-null    int64  
 2   Maxpulse  169 non-null    int64  
 3   Calories  164 non-null    float64
dtypes: float64(1), int64(3)
memory usage: 5.4 KB
None


Result Explained
The result tells us there are 169 rows and 4 columns


  RangeIndex: 169 entries, 0 to 168
  Data columns (total 4 colu
Kết quả được giải thích
Kết quả cho chúng ta biết có 169 hàng và 4 cột


  RangeIndex: 169 mục, 0 đến 168
  Cột dữ liệu (tổng cộng 4 cột):mns):

In [ ]:
And the name of each column, with the data type:
   #   Column    Non-Null Count  Dtype  
  ---  ------    --------------  -----  
   0   Duration  169 non-null    int64  
   1   Pulse     169 non-null    int64  
   2   Maxpulse  169 non-null    int64  
   3   Calories  164 non-null    float64
Và tên từng cột, kèm theo kiểu dữ liệu:
   # Cột Dtype không có giá trị rỗng  
  --- ------ -------------- -----  
   0 Thời lượng 169 không null int64  
   1 Xung 169 không null int64  
   2 Maxpulse 169 không null int64  
   3 Calo 164 không null64

Null Values
The info() method also tells us how many Non-Null values there are present in each column, and in our data set it seems like there are 164 of 169 Non-Null values in the "Calories" colu.
Which means that there are 5 rows with no value at all, in the "Calories" column, for whatever rsonn

Empty values, or Null values, can be bad when analyzing data, and you should consider removing rows with empty values. This is a step towards what is called cleaning data, and you will learn more about that in the next terschap
Giá trị rỗng
Phươngc này i nfo()cũng cho chúng ta biết có bao nhiêu giá trị Không Null trong mỗi cột và trong tập dữ liệu của chúng ta, có vẻ như có 164 trong số 169 giá trị Không Null trong cột "ao".

Điều đó có nghĩa là có 5 hàng không có giá trị nào trong cột "Calo", vì bất kỳ gìdo g.

Các giá trị trống hoặc giá trị Null có thể không tốt khi phân tích dữ liệu và bạn nên cân nhắc việc xóa các hàng có giá trị trống. Đây là một bước tiến tới cái được gọi là làm sạch dữ liệu và bạn sẽ tìm hiểu thêm về điều đó trong các chương teo.ters.